src/test/java/io/github/sskorol/converters/PhoneNumberProcessor.java

```diff
@@ -0,0 +1,10 @@
+package io.github.sskorol.converters;
+
+import com.creditdatamw.zerocell.converter.Converter;
+
+public class PhoneNumberProcessor implements Converter<String> {
+    @Override
+    public String convert(String value, String s1, int i) {
```

Comment:
Let's use the same variables' naming as in the original [interface](https://github.com/creditdatamw/zerocell/blob/master/core/src/main/java/com/creditdatamw/zerocell/converter/Converter.java#L8).

The same comment is for all the other converters.

Reply:
In older version, the parameters names were not convincing. I will fix this.